### ISSUE TITLE
CollectionView not adding collection listeners

### ISSUE THREAD

**Issue by jeff-h** [marionettejs/backbone.marionette#3015]
I'm using Mn 3.0.0-pre.3 (with Backbone 1.3.2), and finding that my CollectionView is not setting up its collection event listeners (and thus not responding to changes on the collection).

I'm instantiating like so:

```js
    this.participantsCollection = new ParticipantsCollection();

    this.listView = new ListView({
      collection: this.participantsCollection,
    });
```

where `ListView` is simply:

```js
  var ListView = Marionette.CollectionView.extend({
    // Nothing relevant here
  });
```

The problem: **in the CollectionView's `_initialEvents`, the event listeners are added only if `this.collection` is true. In my case, when `_initialEvents` is run, there's no `this.collection` so it does nothing.**

In the CollectionView's constructor (https://github.com/marionettejs/backbone.marionette/blob/next/src/collection-view.js#L27-L42) `this.collection` isn't set (for me anyway) until *after* 

```js
Backbone.View.prototype.constructor.call(this, this.options);
```
has run.

I checked Mn 2.4.5 and its constructor has the same flow, so I don't understand how this works for everyone else, but not me! Help and ideas gratefully accepted!

**Comment by paulfalgout** [comment]
I suspect you're running the minified version of Bb `1.3.2`?  That one has a broken `once` function.  It was fixed in `1.3.2`, but the minified file still matched `1.3.1` which contains the bug.  The unminified version of `1.3.2` or either of `1.3.3` will fix the problem (although npm won't work currently with Bb `1.3.3`)

Here's a fiddle with it working: https://jsfiddle.net/q82y91d7/1/  If you swap out bb to 1.3.2 min if will break as you've suggested.

**Comment by jeff-h** [comment]
Wow, you're absolutely right on all fronts. Thanks so much for taking the time to reply.
